src/components/inputs/workspace-launcher/workspace-launcher.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { showSnackbar } from '@openmrs/esm-framework';
+import { useLaunchWorkspaceRequiringVisit } from '@openmrs/esm-patient-common-lib/src/useLaunchWorkspaceRequiringVisit';
```

Comment:
For consistency:

```suggestion
import { useLaunchWorkspaceRequiringVisit } from '@openmrs/esm-patient-common-lib';
```

Reply:
This is the recommended way in cases where we need imports from a single source rather than the whole module for faster optimization. Similar usages have been used in other packages within 3.x for example in the medication s [HERE](https://github.com/openmrs/openmrs-esm-patient-chart/blob/6e126a5be1183f911d68187acb2aacdcabe72edb/packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx#L23)